.coderabbit.yaml

```
@@ -0,0 +1,18 @@
+language: ko-KR
+tone_instructions: "1. 리뷰 시에는 변경 사항의 문제점이나 한계점을 구체적으로 짚어주고, \n 2. 왜 해당 부분이 비효율적이거나 권장되지 않는지 명확한 근거와 함께 설명해주세요. \n 3. 대안을 제시할 땐, 선택지를 제안하면서 각각의 장단점도 논리적으로 설명해주세요. \n 4. 단순 지적보다는 개선 방향에 집중하고, 학습적인 관점에서 이해를 돕는 코멘트를 남겨주세요. \n 5. 지나치게 딱딱하지 않되, 논리의 흐름이 분명하게 전달되도록 해주세요."
+
+early_access: false             # 베타 기능 미사용
+reviews:
+  profile: assertive             # 단도직입적
+  request_changes_workflow: false # change 요청 시 워크플로우 비활성화
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)